client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-preview-auto-height.tsx

```diff
@@ -57,7 +57,7 @@ const PatternPreviewAutoHeight = ( {
 			aria-hidden
 			tabIndex={ -1 }
 			style={ {
-				height,
+				height: height + 2,
```

Comment:
Could you add a comment for this one?

Reply:
Sure 👍  Some patterns need an extra 2px on the height to avoid the scrollbars in the iframe.